client/src/styles/legacy/_uitheme.scss

```diff
@@ -16,10 +16,15 @@
 }
 
 .ui-widget-content,
-.ui-widget {
+.ui-widget,
+.ui-widget.ui-widget-content {
   color: $body-color;
   font-size: 1em;
   border: 0;
+
+  a {
+    color: $link-color;
```

Comment:
These CSS changes are effectively to revert some CSS additions that went into later versions of jquery-ui